data/events/scripts/player.lua

```diff
@@ -86,6 +86,25 @@ function Player:onLookInShop(itemType, count)
 end
 
 function Player:onMoveItem(item, count, fromPosition, toPosition, fromCylinder, toCylinder)
+	-- Store Inbox
```

Comment:
I semi agree with you, as I have said, if a movement script bugged that the player now uses it should bug.
If there is a problem with the store inbox it should bug.

Having 50% of a logic working dosnt seem like a good ide to me.
But it might be better if we ask @djarek and @ranisalt